docs/Getting-Started.md

```diff
@@ -0,0 +1,178 @@
+        return {
+          store: todoStore,
+          listener: function () {
+            this.setState({ todos: todoStore.getAll() });
+          }
+        };
+      },
+
+      registerOwnerProps: function () {
+        return {
+          add: todoActions.add,
+          remove: todoActions.remove
+        };
+      },
+
+      render: function () {
+        var Todos = this.state.todos.map(function(todo) {
+          return (
+            <Todo todo={todo} />
+          );
+        });
+
+        return (
+          <div>
+            <TodoCreateForm />
+            {Todos}
+          </div>
+        );
+      }
+    });
+```
+
+`Tux` also provides the ability to build high performance components out of the box. In our Todo application within the individual Todo component, a Todo's text is a `mutableTraits` property. Additionally the Mutable class is an Ownee component, so it allows you to validate the props that have been passed down to that individual component using `nearestOwnerPropTypes`.
+
+A high performance component:
+
+```javascript
+    var React = require('tux/React');
+
+    var Todo = React.createMutableClass({
+      mutableTraits: {
+        props: 'text'
+      },
+
+      propTypes: {
+        todo: React.PropTypes.object.isRequired
+      },
+
+      nearestOwnerPropTypes: {
+        remove: React.PropTypes.func.isRequired
+      },
+
+      handleRemove: function (e) {
+        e.preventDefault();
+        this.nearestOwnerProps.remove(this.props.todo);
+      },
+
+      render: function () {
+        return (
+          <p>{ this.props.todo.text }</p>
+          <button onClick={this.handleRemove}>Delete</buttion>
+        );
+      }
+    });
+
+    module.exports = Todo;
+```
+
+Another of the opinionated `Tux` components are the Ownee class. These components don't manage state, unless it is needed for input form validation. They don't need to manage their own state standardly as they receive their static methods and props from their closest Owner component. The `TodoCreateForm` utilizes this by accessing the `add` method from its nearest owner component, which is the `TodoViewOwner`.
+
+```javascript
+    var TodoCreateForm = React.createOwneeClass({
+      nearestOwnerPropTypes: {
+        add: React.PropTypes.func.isRequired
+      },
+
+      handleSubmit: function (e) {
+        e.preventDefault();
+        var todoTextInput = this.refs.textInput.getDOMNode();
+
+        this.nearestOwnerProps.add({
+          text: todoTextInput.value
+        });
+
+        todoTextInput.value = '';
+      },
+
+      render: function(){
+        return (
+          <form onSubmit={this.handleSubmit}>
+            <input type="text" ref="textInput" placeholder="Add a Todo" />
+            <button type="submit">Add</button>
+          </form>
+        );
+      }
+    });
+
+    module.exports = TodoCreateForm;
+```
+
+Tux abstracts away the complexity of Flux with powerful `Actions` syntax:
```

Comment:
Need to explain what is happening here more:

`Tux` leverages the `Flux` architecture but abstracts away all of the normal boilerplate associated with `Flux`. In `Tux` we build `actionCategories` and then we can invoke them in order to dispatch their actions and register with them in order to receive these actions. Here is the `todoActions` category that we required in from before.